apps/sms/style/root.css

```diff
@@ -177,3 +177,11 @@ menu, nav, section, video, audio, canvas, progress, meter, time {
   margin: 0;
 }
 
+/* Dialogs */
+body.dialog-animating {
+  pointer-events: none;
+}
+
+body.dialog-animating .disable-when-dialog-animates {
+  pointer-events: none;
+}
```

Comment:
I think this should be in sms.css; root.css is like a "reset.css" file in my opinion